setup.py

```diff
@@ -134,6 +130,7 @@ def get_winver():
 
 def main():
     setup_args = dict(
+        zip_safe=False,
```

Comment:
According to this:
https://pythonhosted.org/setuptools/setuptools.html#setting-the-zip-safe-flag
zip_safe=False seems to be unnecessary because if the package contains C extension files (as psutil) zip_safe is already implicitly set to False. Given that zip_safe is an option which exists only if setuptools is installed I think it makes sense to remove it.